www/content/release.md

```diff
@@ -45,6 +45,13 @@ release:
   # GitHub.
   # Defaults to false.
   disable: true
+
+  # You can add extra pre-existing files to the release.
+  # The left side is the name you want on the github release and the right
+  # side is the path to the file.
+  # Defaults to empty.
+  extra_files:
+    name_on_github: ./path/to/file.txt
```

Comment:
have you considered something like doublestar glob syntax?

the equivalent feature on both GitHub workflows and Travis CI offer `files:` field where you can just glob like `**/**.bar`

Reply:
yes, I'm worried about that because we would have to guess the filename on github side then... 

we could use the last part of the path, but it could end up with duplicated names - which would lead to failed releases 🤔

eg:

```
foo/bar/1.md
foo/lala/1.md
```

```
extra_files:
- foo/**/*.md
```

in this case we would get `1.md` 2 times 🤔

I'm not sure if this is something we can just leave to the user (and accept the globs) or be more restrict about (current impl)...

Reply:
gently ping @ahmetb

Reply:
oops sorry.

Most extensible and fail-safe way would be to fail maybe when you get name overlaps.

Similarly you can use something like:

```
extra_files:
- file: foo
- file: bar
- file_glob: foo/**/**.tar.gz
```

this way the user can control the behavior.

Reply:
yeah I guess it makes sense, done.

Reply:
I assume you went for extra_files_globs, which isn't exactly what I suggested but if you see that as fit that's ok.